modules/dynamic-plugins/con-dynamic-plugins-cache.adoc

```diff
@@ -0,0 +1,66 @@
+[id="con-dynamic-plugin-cache_{context}"]
+
+= Using the dynamic plugins cache
+The dynamic plugins cache in {product} ({product-very-short}) enhances the installation process and reduces platform boot time by storing previously installed plugins. If the configuration remains unchanged, this feature prevents the need to re-download plugins on subsequent boots.
+
+When you enable dynamic plugins cache:
+
+* The system calculates a checksum of each plugin's YAML configuration (excluding `pluginConfig`).
+* The checksum is stored in a file named `dynamic-plugin-config.hash` within the plugin's directory.
+* During boot, if a plugin's package reference matches the previous installation and the checksum is unchanged, the download is skipped.
+* Plugins that are disabled since the previous boot are automatically removed.
+
+== Enabling the dynamic plugins cache
+To enable the dynamic plugins cache in RHDH, the plugins directory `dynamic-plugins-root` must be a persistent volume. 
+
+For Helm chart installations, a persistent volume named `dynamic-plugins-root` is automatically created.
+
+For operator-based installations, you must manually create the PersistentVolumeClaim (PVC) as follows:
+
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-root
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+
+---
+
+apiVersion: rhdh.redhat.com/v1alpha2
+kind: Backstage
+metadata:
+  name: rhdh
```

Comment:
```suggestion
  name: developer-hub
```
with rhdh name it will create a new instance instead of applying the changes to the default developer-hub instance